engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkModelJar.scala

```diff
@@ -34,14 +36,29 @@ class FlinkModelJar extends LazyLogging {
     tempFile
   }
 
+  private def additionalArtifactsToIncludeInJar(): List[URL] = {
```

Comment:
I thought about an idea that we explicitly add `flink-additional-deps` into `modelConfig/classPath` in our `application.conf` and `dev-application.conf` in our distribution and here have a `if classpath doesn't contain flink-additional-deps then add them` code. Thanks to that this code will be only needed in the transitive period and after some time, we could remove it and stay with the explicit approach. WDYT?

Reply:
sounds like a good idea, I will make it work like that